natlas-server/requirements.txt

```diff
@@ -27,6 +30,7 @@ python-editor==1.0.4
 python-libnmap==0.7.0
 semver==2.9.0
 sentry-sdk[flask]==0.10.2
+setuptools==28.8.0
```

Comment:
Is this an actual dependency? Was it brought in by opencensus? As long as it's being installed intentionally then this comment can be resolved.

Reply:
I did install it intentionally. I was encountering some Docker image build issues without. I'll go check to see if it's still an issue.

Reply:
```
er, sentry-sdk, sympy
  Running setup.py install for SQLAlchemy: started
    Running setup.py install for SQLAlchemy: finished with status 'done'
  Running setup.py install for Mako: started
    Running setup.py install for Mako: finished with status 'done'
  Running setup.py install for alembic: started
    Running setup.py install for alembic: finished with status 'done'
  Running setup.py install for blinker: started
    Running setup.py install for blinker: finished with status 'done'
  Running setup.py install for Flask-Login: started
    Running setup.py install for Flask-Login: finished with status 'done'
  Running setup.py install for Flask-Mail: started
    Running setup.py install for Flask-Mail: finished with status 'done'
  Running setup.py install for mpmath: started
    Running setup.py install for mpmath: finished with status 'done'
  Running setup.py install for contextvars: started
    Running setup.py install for contextvars: finished with status 'done'
  Found existing installation: setuptools 28.8.0
    Uninstalling setuptools-28.8.0:
      Successfully uninstalled setuptools-28.8.0
  Running setup.py install for googleapis-common-protos: started
    Running setup.py install for googleapis-common-protos: finished with status 'error'
    Complete output from command /usr/local/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-yqs6w4x0/googleapis-common-protos/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-jpn6ct04-record/install-record.txt --single-version-externally-managed --root /install --compile:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
    ModuleNotFoundError: No module named 'setuptools'
```